database/cassandra/schema.cql

```diff
@@ -0,0 +1,22 @@
+CREATE KEYSPACE grubhub
+    WITH REPLICATION = {'class': 'SimpleStrategy', 'replication_factor' : 1};
+
+USE grubhub;
+
+CREATE TABLE menu_items (
+  itemId int PRIMARY KEY,
+  restaurantId int,
+  category text,
+  name text,
+  price decimal,
+  description text,
+  pictureUrl text,
+  popular boolean,
+  spicy boolean,
+  required text,
+  optional text
+);
+
+COPY menu_items(itemId,restaurantId,category,name,price,description,pictureUrl,popular,spicy,required,optional) 
+    FROM '/Users/phdickson/Desktop/SDC/FEC-menu/database/cassandra/dataGenerationScripts/menuItemsCassandra.csv' 
```

Comment:
I think it's best practice if you hide the entire filepath to your `.csv` file and not make it public.

Reply:
And instead write a `FILL_ME_IN`flag.

Reply:
Thanks for the tip. I will change that.